src/Cli/dotnet/Commands/Solution/Add/SolutionAddCommand.cs

```diff
@@ -193,14 +194,14 @@ private void AddProject(SolutionModel solution, string fullProjectPath, ISolutio
         foreach (var solutionPlatform in solution.Platforms)
         {
             var projectPlatform = projectInstancePlatforms.FirstOrDefault(
-                platform => platform.Replace(" ", string.Empty) == solutionPlatform.Replace(" ", string.Empty), projectInstancePlatforms.FirstOrDefault());
+                platform => platform.Replace(" ", string.Empty) == solutionPlatform.Replace(" ", string.Empty), projectInstancePlatforms.FirstOrDefault()!);
```

Comment:
Lacking the `!` on the default value (last argument) the compiler will end up inferring `FirstOrDefault<string?>` not `FirstOrDefault<string>`. That is because the default value directly infers the value for `T` and `FirstOrDefault` is `null` returning. Don't know enough about the semantics here to suggest a more null friendly pattern.